rhel8/profiles/ncp.profile

```diff
@@ -177,7 +177,7 @@ selections:
 
     ### Module Blacklist
```

Comment:
The `coreos-ncp.profile` needs to stay in the `ocp4` product directory.

Reply:
@evgenyz can you please make this change and then take out of draft mode?

Reply:
It won't be build-able with only this change

Reply:
@evgenyz understood, but this PR is going to start holding up ocp4 content enablement soon.

Reply:
This profile uses rules that are not compatible with 'ocp4' product. And I don't really like the idea of adding 'ocp4' as a product to all these rules in `linux_os/guide`, because 'ocp4' is not in fact Linux OS. 

Right now I'm trying to add a way to list multiple `benchmark_root` elements with a sane result, and the deeper I'm going into the problem less and less it is clear for me what is the actual benefit.

Reply:
We think that this could be solved by temporarily setting documentation_complete to false until the other PR is merged.